packages/blockchain-extension/extension/commands/deleteEnvironmentCommand.ts

```diff
@@ -101,6 +101,9 @@ export async function deleteEnvironment(environment: FabricEnvironmentTreeItem |
 
         await vscode.workspace.getConfiguration().update(SettingConfigurations.FABRIC_RUNTIME, localSettings, vscode.ConfigurationTarget.Global);
 
+        await vscode.commands.executeCommand(ExtensionCommands.REFRESH_GATEWAYS);
```

Comment:
you don't need this. When an environment is deleted it will automatically refresh the gateways and wallets